resources/definitions/zone3d_printer.def.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "zone3d_printer",
+    "name": "Zone3d Printer",
```

Comment:
I can't find this printer on the web, but is it logical to have the word "Printer" there?